Server/User/routers/bookings.py

```
@@ -468,3 +568,39 @@ async def complete_consultation(
         message="Consultation completed successfully",
         booking=booking_response
     )
+
+
+@router.post("/notifications/create")
+async def create_notification(
+    request: dict,
+    db: Session = Depends(get_db)
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)